src/App.tsx

```diff
@@ -37,6 +38,7 @@ const App = () => {
           <Toast />
           <CssBaseline />
           <IdenfyPingUserStatus />
+          <IdenfyModal />
```

Comment:
I get why you added this to the global app, but i think it might be an overkill. With this, it would run some logic on every single page load, even after the user is verified. I think we should only add the component where needed at the moment it is only in 2 places, even if it was in 5 i dont think the performance hit is worth putting it at the app level. Thoughts?